src/components/section/header.jsx

```
@@ -1,82 +1,52 @@
-                        </li>
-                    ))}
-                </ul>
-            </div>
-        </header>
-    );
-};
+  return (
+    <div className='header2'>
+      <div className="header__content">
+        <a className='header__logo' href="/">
+            <img className='logo' src='/img/Logo1.png'></img>
```

Biome (2.1.2)

[error] Provide a text alternative through the alt, aria-label or aria-labelledby attribute

Meaningful alternative text on elements helps users relying on screen readers to understand content's purpose within a page.
If the content is decorative, redundant, or obscured, consider hiding it from assistive technologies with the aria-hidden attribute.

(lint/a11y/useAltText)